# Potencia en un sistema trifásico balanceado

__UNLZ - Facultad de Ingeniería__       
__Electrotecnia__      
__Alumno:__ Daniel Antonio Lorenzo

<a href="https://colab.research.google.com/github/daniel-lorenzo/Electrotecnia/blob/master/Potencia_trifasico.ipynb"><img align="left" src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open in Colab" title="Open and Execute in Google Colaboratory"></a>

## Ejemplo 12.6
Determine los valores totales de la potencia promedio, potencia reactiva y potencia compleja en la fuente y en la carga.

<img src="img/ej12-2.png">

### Solución
Es suficiente considerar una fase, ya que el sistema está balanceado. En relación con la fase $a$.

In [1]:
import math, cmath

In [2]:
# Datos:
Zy = 5-2j + 10+8j
Vp = 110

Ip = Vp/Zy

In [3]:
print('Zy = {:.0f} Ohm'.format(Zy))
print('Ip = (%.2f < %.2f°) A'%(abs(Ip) , math.degrees( cmath.phase(Ip) ) ))

Zy = 15+6j Ohm
Ip = (6.81 < -21.80°) A


$$ V_p = 110 \angle 0^\circ \, \mathrm{V} \quad \mbox{y} \quad I_p = 6,81 \angle -21,8^\circ $$

Así, en la fuente, la potencia compleja suministrada es

$$ S_s = 3 V_p I_p^\ast $$

In [4]:
Ss = 3*Vp*Ip.conjugate()

In [20]:
print('Ss = {:.1f} VA'.format(Ss))

Ss = 2086.2+834.5j VA


En la carga, la potencia compleja absobida es

$$ S_L = 3 |I_p|^2 Z_p $$

In [23]:
Zp = 10+8j

SL = 3*abs(Ip)**2 * Zp

In [24]:
print('SL = {:.1f} VA'.format(SL))

SL = 1390.8+1112.6j VA


La potencia real absorbida es de 1390.8 W y la potencia reactiva absorbida de 1112.6 VAr. La diferencia entre las dos potencias complejas es absorbida por la impedancia de línea $(5-j2) \, \Omega$. Para demostrar que éste es el caso, la potencia compleja absorbida por la línea se halla como

$$ S_\ell = 3|I_p|^2 Z_\ell $$

In [25]:
Zl = 5-2j # Ohm

Sl = 3*abs(Ip)**2 * Zl

In [26]:
print('Sl = {:.1f} VA'.format(Sl))

Sl = 695.4-278.2j VA


la cual es la diferencia entre $S_s$ y $S_L$, como era de esperar.

In [18]:
print('Ss - Sl = {:.1f} VA'.format(Ss - SL) )

Ss - Sl = 695.4-278.2j VA


In [27]:
%reset -s -f

## Problema de práctica 12.6
En referencia al circuito Y-Y del problema de práctica 12.2, calcule la potencia compleja en la fuente y en la carga.

<img src="img/ejp12-2.png">

In [28]:
import math, cmath

In [29]:
# Datos:
Zy = (0.4+0.3j) + (0.6+0.7j) + (24+19j) # Ohm
Zp = 24+19j # Ohm
Vp = cmath.rect(120, math.radians(30) )

In [ ]:
Ip = Vp/Zy

$$ I_p = \frac{V_p}{Z_Y} $$

In [30]:
print('Zy = {:.0f} Ohm'.format(Zy))
print('Ip = (%.2f < %.2f°) A'%(abs(Ip) , math.degrees( cmath.phase(Ip) ) ))

Zy = 25+20j Ohm
Ip = (3.75 < -8.66°) A


In [31]:
# Potencia compleja suministrada
Ss = 3*Vp*Ip.conjugate()

$$ S_s = 3 V_p I_p^\ast $$

In [36]:
print('---------------------')
print('Ss = {:.1f} VA'.format(Ss))
print('---------------------')

---------------------
Ss = 1053.7+842.9j VA
---------------------


In [33]:
# Potencia compleja absorbida
SL = 3*abs(Ip)**2*Zp

$$ S_L = 3 |I_p|^2 Z_p $$

In [37]:
print('---------------------')
print('SL = {:.1f} VA'.format(SL))
print('---------------------')

---------------------
SL = 1011.5+800.8j VA
---------------------


In [38]:
%reset -s -f

## Ejemplo 12.7
Un motor trifásico puede considerarse una carga en Y balanceada. Un motor trifásico toma 5.6kW cuando la tensión de línea es de 220 V y la corriente de línea de 18.2 A. Determine el factor de potencia del motor.

### Solución
La potencia es
$$ S = \sqrt{3} V_L I_L $$

Dado que la potencia real es
$$ P = 5600 \, \mathrm{W} = S \cos \theta $$

el factor de potencia
$$ \mathrm{fp} = \cos \theta = \frac{P}{S} $$

In [40]:
import math, cmath

In [41]:
# Datos
VL = 220 # V
IL = 18.2 # A
P = 5600 # W

In [42]:
S = math.sqrt(3)*VL*IL
fp = P/S

In [45]:
print('Resultados:')
print('S = %.2f VA'%S)
print('fp = %.4f'%fp)

Resultados:
S = 6935.13 VA
fp = 0.8075


In [46]:
%reset -s -f

## Problema de práctica 12.7
Calcule la corriente de línea requerida para un motor de 30kW con un factor de potencia atrasado de 0.85 si se conecta a una fuente balanceada con una tensión de línea de 440 V.

In [47]:
import math, cmath

In [48]:
# Datos:
P = 30e3 # W
fp = 0.85
VL = 440 # V

el factor de potencia
$$ \mathrm{fp} = \cos \theta = \frac{P}{S} $$

Entonces
$$ S = \frac{P}{\mathrm{fp}} $$

In [49]:
S = P/fp

La potencia es
$$ S = \sqrt{3} V_L I_L $$

Entonces
$$ I_L = \frac{S}{\sqrt{3} V_L} $$

In [50]:
IL = S/(math.sqrt(3)*VL)

In [51]:
print('IL = %.2f A'%IL)

IL = 46.31 A


In [52]:
%reset -s -f

-----------
<a href="https://colab.research.google.com/github/daniel-lorenzo/Electrotecnia/blob/master/Potencia_trifasico.ipynb"><img align="left" src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open in Colab" title="Open and Execute in Google Colaboratory"></a>